sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/client/CachedSchemaRegistryClientBuilder.java

```diff
@@ -74,6 +76,10 @@ public CachedSchemaRegistryClientBuilder() {
         this.credential = null;
         this.retryPolicy = new RetryPolicy("retry-after-ms", ChronoUnit.MILLIS);
 
+        HttpHeaders headers = new HttpHeaders();
+        headers.put("api-version", "2017-04");
```

Comment:
The version usually is the year and month when the service is released. Why is this set to 2017? Also, to set the service version, use `ServiceVersion` interface and define an enum like this - [ConfigurationClientBuilder](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java#L381)

Reply:
We will make this configurable in the next release.

Reply:
Adding todo in my local branch.